src/scripts/main.js

```diff
@@ -1,8 +1,21 @@
 'use strict';
 
 const people = require('./lib/people');
+const dashboard = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
+  const tableRow = document.createElement('tr');
+  const gender = (person.sex === 'm' ? 'Male' : 'Female');
+  const century = Math.ceil(person.died / 100);
 
-// write your code here
+  tableRow.insertAdjacentHTML('beforeend',
+    `<td>${person.name}</td>
+    <td>${gender}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${person.died - person.born}</td>
+    <td>${century}</td>`
```

Comment:
```suggestion
  tableRow.insertAdjacentHTML('beforeend',
    `<td>${person.name}</td>
    <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
    <td>${person.born}</td>
    <td>${person.died}</td>
    <td>${person.died - person.born}</td>
    <td>${Math.ceil(person.died / 100)}</td>`
```

You can write those expressions right here and get rid of redundant variables 'gender' and 'century'